docs.aerogear-site-hosting.md

```diff
@@ -0,0 +1,21 @@
+# Hosting of docs.aerogear.org web site
+
+## Introduction
+
+Previously all AeroGear docs were hosted on a file server and associated with the domain. We'd like to take advantage of github for hosting for reliability and ease of use.
+
+## Problem Description
+
+Hosting html (and associated files) for docs.aerogear.org is essential for communication with users. Need to automatically publish any changes to mobile-docs repo to docs.aerogear.org.
+
+## Expectations
+
+- If we take an approach used by https://github.com/debezium/debezium.github.io where asciidoc and html files are both in same repo (adoc in develop branch, html in master branch), we can anticipate the repo getting large over time with history of html changes mixed with history of adoc file changes.
+
+  
+## Proposed Solution
+
+Create a docs.aerogear.github.io repo and use it to host docs.aerogear.org html files.
+
+Use a CI service (eg circleci) to run asciibinder and push the output to the docs.aerogear.github.io repo.
```

Comment:
Worth to state that this will rebuild master.
Problem is that rebuild of the webpage docs needs to be aligned with the changes that caused docs change.
It's best to have specific branch that will be updated and tagged for release.

We hit this hard in RainCatcher - some cutting edge changes were merged into docs without release available.

Reply:
@wtrocki I'm thinking master, and have updated doc with my expectations, that there won't be 'release' of mobile.next upstream, therefore there won't be a version branch or tag